cycode/cli/commands/scan/code_scanner.py

```diff
@@ -560,10 +580,13 @@ def print_debug_scan_details(scan_details_response: 'ScanDetailsResponse') -> No
 
 
 def print_results(
-    context: click.Context, local_scan_results: List[LocalScanResult], errors: Optional[Dict[str, 'CliError']] = None
+    context: click.Context,
+    local_scan_results: List[LocalScanResult],
+    errors: Optional[Dict[str, 'CliError']] = None,
+    aggregation_report_url: str = '',
```

Comment:
we could remove it from here if we save the aggregation report url to the context. otherwise, should it be `None` by default instead of empty string? and type hint is `Optional[str]`